sdk/keyvault/keyvault-common/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@azure/keyvault-common",
+  "sideEffects": false,
+  "private": true,
+  "author": "Microsoft Corporation",
+  "version": "1.0.0",
+  "license": "MIT",
+  "description": "Isomorphic client library for Azure KeyVault's certificates.",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "main": "./src/index.ts",
+  "module": "dist-esm/index.js",
+  "types": "./types/index.d.ts",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "scripts": {
+    "audit": "node ../../../keyvault-common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
```

Comment:
feels weird that it should go three levels up to come back down to itself

Reply:
I can't think of how to make this better at the moment.

Reply:
am I missing something? or can this line not be `node ./scripts/rush-audit.js` instead?

Reply:
oh this is a mistake, it should be common/ not keyvault-common/ Thank you!

Reply:
(this is not part of keyvault-common)

Reply:
I'll skip this script for now